src/NovaDependencyContainer.php

```diff
@@ -153,6 +153,23 @@ public function resolveForDisplay($resource, $attribute = null)
                 continue;
             }
 
+            if (is_object($resource->{$dependency['property']}) && is_array($dependency['value'])) {
+                if (array_key_exists('value', $dependency) && in_array($resource->{$dependency['property']}->id, $dependency['value'])) {
+                    $this->meta['dependencies'][$index]['satisfied'] = true;
+                    continue;
+                }
+            }
+
+            if (is_iterable($resource->{$dependency['property']})) {
```

Comment:
Could you give an example in which scenario `is_iterable($resource->{$dependency['property']})` is used?

Reply:
This should match the other condition for when you want to match a field value to be within an array of items.